docs/modules/operation/pages/geographical-map/introduction.adoc

```diff
@@ -0,0 +1,36 @@
+OpenNMS includes a geographical map which can show nodes with address asset information on a world map, including dynamic alarm status.
+
+The geographical map is located in the `Maps` menu or by going to the page of a node that has an address and/or longitude/latitude defined in its asset fields, and following the `View in Geographical Map` link.
+
+## Markers and Clusters
+
+The geographical maps display each node that has an address, or longitude and latitude in the database. When the geographical map is loaded, it will look at each node. If the node has coordinates, it will display it on the map. If the node has an address, it will use a geolocation service to convert the address to coordinates, and then display it on the map.
+
+The geographical maps will automatically cluster nodes into groups if they are close together. By hovering over a cluster (the circle marker with a number in it), it will show the boundaries of the nodes it contains. Click on a marker to show the summary of the nodes it contains.
+
+## Status
+
+A node's marker will display as the color associated with the most severe alarm for that node.  A cluster's marker will display as the color of the highest severity of the nodes the marker contains.
+
```

Comment:
Unfortunately, the images are not showing up properly when I build the file locally. I've tried modifying the syntax, but nothing is working. It is possible to put images in tables, so we'll have to figure out what's going wrong.

Reply:
@Bonrob2 Maybe a stupid question but how do you "build" the docs locally? Do we have a defined procedure for that?
In our company, we add readme files into folders like https://github.com/OpenNMS/opennms/tree/develop/docs with usage/build instructions. Maybe it makes sense to add a readme for this? Otherwise, nobody is able to give it a try, right?

Reply:
That's a good idea. I'll add a ticket for that.

Reply:
So how can I give it locally on my computer a try to test, if or why the images are not showing up?
I've read that images in tables can be tricky. Maybe also not worth doing it like this?
As @mmahacek mentioned we could also drop the table and just add the images with titles or similar. Wdyt?

Reply:
I removed the table and just put the picture and a "title" above.